porthtml.html

```diff
@@ -78,15 +78,152 @@ <h6 class="LET"><a href="#">LET’S CONNECT</a></h6>
          
           
         </ul>
+        </div>
+        
         <img src="abc.svg" alt="" width="20" height="20">
 
         <img src="rec.svg" alt="" width="20" height="20">
 
+      
+
+    <div id="modal" class="modal modal--active">
+         
+        
+      <div>
+      <img class="whole-card" src="card2.svg" alt="Work preview for project Uber Navigation">
+      </div>
+      <br>
+      <div>
+      <img class="whole-card" src="card1.svg" alt="Work preview for project Uber Navigation">
+      </div>
+      <br>
+      <div>
+        <img class="whole-card" src="card3.svg" alt="Work preview for project Uber Navigation">
+        </div>
+        <br>
+        <div>
+          <img class="whole-card" src="card4.svg" alt="Work preview for project Uber Navigation">
+          </div>
+        
+        <div class="modal__info">
+          <p class="modal__desc">Lorem ipsum dolor sit amet consectetur adipisicing elit. Veniam architecto sint, quae
+            sit nostrum accusamus magnam blanditiis iusto in ducimus eligendi voluptatibus commodi.</p>
+          <div class="modal__tags-buttons-container">
+            <ul class="modal__tech">
+           
+            </ul>
+            <div class="modal__line"></div>
+            <div class="modal__buttons-container">
+              <a href="#" class="modal__button modal__button--see-live" target="_blank">See live<i
+                  class="modal__icon-button fas fa-external-link-alt"></i></a>
+              <a href="#" class="modal__button modal__button--see-src" target="_blank">See source<i
+                  class="modal__icon-button fab fa-github"></i></a>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] There is no need to add unnecessary code, it makes the website unclean. Let's remove it.